analytics.ts

```diff
@@ -132,6 +133,9 @@ export const inject = (deps: UsageAnalyticsDeps) => {
 					taqVersion: parsedArgs.setVersion,
 					operatingSystem: machineInfo.os,
 					fetch,
+					fields: {
```

Comment:
Is `fields` treated specially by Google Analytics? Can we just flatten `taq_ui` instead of having it inside `fields`?

Reply:
`fields` is a key/value mapping of event fields that you want to include in all events that are added/tracked. Think of them as default or global fields that you want as part of the event payload regardless of what event you sent.

All key/value pairs in fields get merged with the event data in the addEvent function:
https://github.com/ecadlabs/taqueria/blob/vscode-analytics/taqueria-analytics/index.ts#L32